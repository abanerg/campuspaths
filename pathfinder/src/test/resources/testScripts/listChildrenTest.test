# Create graph

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D

AddEdge graph A B 10.000
AddEdge graph A C 10.000
AddEdge graph A C 5.000
AddEdge graph A D 1.000

# Print children

ListChildren graph A
