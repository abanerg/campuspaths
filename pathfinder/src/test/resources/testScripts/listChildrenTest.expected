# Create graph

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph

added edge 10.000 from A to B in graph
added edge 10.000 from A to C in graph
added edge 5.000 from A to C in graph
added edge 1.000 from A to D in graph

# Print children

the children of A in graph are: B(10.000) C(5.000) C(10.000) D(1.000)
